### PR TITLE
add dog entity, facade, controller

### PR DESCRIPTION
**Pull request by tjddnghkrk**
#### What
- add dog entity

* Feat : add dog controller
* Feat : add dog facade

### CODE REVIEW

**Comment by tjddnghkrk** [comment]
add dog entity
Feat : add dog controller
Feat : add dog facade

File: src/main/java/com/soosoo/soosoo/controller/dog/DogController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.soosoo.soosoo.controller.dog;
+
+import com.soosoo.soosoo.service.dog.facade.DogFacade;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("notice")
```
Comment (kor-kms) [inline src/main/java/com/soosoo/soosoo/controller/dog/DogController.java:9]: notice -> dog

File: src/main/java/com/soosoo/soosoo/controller/dog/DogController.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@RequiredArgsConstructor
+public class DogController {
+    private final DogFacade noticeFacade;
```
Comment (kor-kms) [inline src/main/java/com/soosoo/soosoo/controller/dog/DogController.java:12]: noticeFacade -> dogFacade

**Comment by tjddnghkrk** [comment]
naming 수정하겠습니다!

**Pull request closed without merging**